inorbit_edge/utils.py

```diff
@@ -0,0 +1,46 @@
+import math
```

Comment:
nit. `utils.py` is a very generic name, do you think we can use something more specific ?